CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-01-15
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* Starting in API version 2023-01-01, the following API changes will occur:
+  * The name for the AKS cluster SKU Name `Basic` will change to `Base`. `Basic` will be removed in the 2023-07-01 API version.
+  * The name for the AKS cluster SKU Tier `Paid` will change to `Standard`. `Paid` will be removed in the 2023-07-01 API version.
+* AKS began pod security policy deprecation on 2022-11-01 API. The [pod security policy](https://learn.microsoft.com/azure/aks/use-pod-security-policies) will be removed completely on 2023-06-01 API with AKS 1.25 version or higher. You can migrate pod security policy to [pod security admission controller](https://learn.microsoft.com/azure/aks/use-psa) before the deprecation deadline.
+
+### Release notes
+
+* Features
+  * Starting with Kubernetes 1.26:
```

Comment:
Having in mind 1.26 is not out yet. This is an announcement in the previous section to be moved here when 1.26 is out in preview/GA?

Reply:
moved